bunner-master/bunner.py

```diff
@@ -172,9 +184,18 @@ def update(self):
                     break
 
             # Are we on the ground, and are there inputs to process?
-            if self.timer == 0:
-                # Take the next input off the queue and process it
-                self.handle_input(self._ai_decide(current_row, next_row))        
+            
+            if state == State.MANUAL:
+            
+                if self.timer == 0 and len(self.input_queue) > 0:
+                    # Take the next input off the queue and process it
+                    self.handle_input(self.input_queue.pop(0))
+            
+            if state == State.AUTO:
+            
+                if self.timer == 0:
+                    # Take the next input off the queue and process it
+                    self.handle_input(self._ai_decide(current_row, next_row))        
 
             land = False
             if self.timer > 0:
```

Comment:
update this chunk so that there is a new state DX and DY need to contain [0,0] coordinate according to 5th new state that means 'stay without moving'